Jenkinsfile

```diff
@@ -97,20 +109,20 @@ def unitTests(os, nodeVersion, npmVersion, testSuiteBranch) {
 									sh 'rm -f mocha_*.crash'
 								} else {
 									// gather crash reports/tombstones for Android
-									sh 'adb -e pull /data/tombstones'
+									sh './adb-all.sh pull /data/tombstones'
```

Comment:
`./adb-all.sh` will run commands on all connected devices and/or running emulators.